coordinator/src/backup.rs

```diff
@@ -0,0 +1,76 @@
+use anyhow::Result;
+use bitcoin::secp256k1::ecdsa::Signature;
+use bitcoin::secp256k1::PublicKey;
+use coordinator_commons::Backup;
+use coordinator_commons::DeleteBackup;
+use coordinator_commons::Restore;
+use sled::Db;
+
+const BACKUPS_DIRECTORY: &str = "user_backups";
+
+/// Holds the user backups in a sled database
+///
+/// TODO(holzeis): This is fine for now, once we grow we should consider moving that into a dedicate
```

Comment:
What's the storage requirement for this?

Reply:
hmm.. hard to say, identifying the storage requirements is an exercise we haven't done yet. But for the small amounts we are currently dealing with I think it doesn't really matter.